readthedocs/restapi/serializers.py

```diff
@@ -43,11 +43,6 @@ class ProjectAdminSerializer(ProjectSerializer):
         slug_field='feature_id',
     )
 
-    show_advertising = serializers.SerializerMethodField()
-
-    def get_show_advertising(self, obj):
-        return obj.show_advertising
-
```

Comment:
I'm unfamiliar with why this change is included here, was this on purpose?

Reply:
Ups, I forgot to explain why I did remove this code.

Basically, it's not necessary. That method is the default behavior of DRF when you put a field name in the `Meta.fields`.

> Alternatively names in the fields options can map to properties or methods which take no arguments that exist on the model class.

From, https://www.django-rest-framework.org/api-guide/serializers/#specifying-which-fields-to-include

Reply:
Roger, I'll add a QA note to our next release.